Broken Peer Dependencies

New Versions of Nativescript `ns create --svelte` ship NS v8.6.1 but `svelte-native` requires `<8.6` as a dependency,

As a result, in every dependency install,  npm yells with 

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: myapp@1.0.0
npm ERR! Found: @nativescript/core@8.6.1
npm ERR! node_modules/@nativescript/core
npm ERR!   @nativescript/core@"~8.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nativescript/core@"~8.5.9" from svelte-native@1.0.15
npm ERR! node_modules/svelte-native
npm ERR!   svelte-native@"1.0.15" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

New developers may not understand the implications of using `--force` or `--legacy-peer-deps` 

Consider supporting both minor and patch releases of NS

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "svelte-native-preprocessor",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "author": "David Pershouse",
   "license": "MIT",
   "repository": {
@@ -16,7 +16,7 @@
   "main": "dist/index.js",
   "types": "types/index.d.ts",
   "peerDependencies": {
-    "svelte": "^3.37.0"
+    "svelte": ">= 3.37.0 < 5"
   },
   "devDependencies": {
     "@types/mocha": "^5.2.6",

